packages/react/cypress/component/auto/form/AutoForm.cy.tsx

```diff
@@ -235,10 +235,8 @@ describeForEachAutoAdapter("AutoForm", ({ name, adapter: { AutoForm }, wrapper }
   });
 
   it("can render a rich text editor for markdown content", async () => {
-    cy.mountWithWrapper(<AutoForm action={api.widget.create} include={["name", "inventoryCount", "description"]} />, wrapper);
+    cy.mountWithWrapper(<AutoForm action={api.widget.create} include={["description"]} />, wrapper);
```

Comment:
Previously, `"name", "inventoryCount"` were only included in this test because they were required. This test passing without those req'd fields proves that those validations don't run on non-shown fields 😎